eng/pipelines/coreclr/perf.yml

```diff
@@ -19,7 +19,7 @@ trigger:
     - SECURITY.md
     - THIRD-PARTY-NOTICES.TXT
   
-pr: none
+#pr: none
```

Comment:
@DrewScoggins, was it intentional to merge it with this line commented out? perf pipeline is triggering in new PRs.

Reply:
No this was an accident, sorry about that.